src/main/java/thePackmaster/cards/intothebreachpack/ReboundVolley.java

```diff
@@ -0,0 +1,57 @@
+package thePackmaster.cards.intothebreachpack;
+
+import com.evacipated.cardcrawl.mod.stslib.cards.interfaces.BranchingUpgradesCard;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.AttackDamageRandomEnemyAction;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.cards.AbstractPackmasterCard;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.atb;
+
+public class ReboundVolley extends AbstractPackmasterCard implements BranchingUpgradesCard {
+    public final static String ID = makeID("ReboundVolley");
+
+    public ReboundVolley() {
+        super(ID, 1, CardType.ATTACK, CardRarity.UNCOMMON, CardTarget.ENEMY);
+        damage = baseDamage = 6;
+        secondDamage = baseSecondDamage = damage;
+        magicNumber = baseMagicNumber = 1;
+    }
+
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        dmg(m, AbstractGameAction.AttackEffect.SLASH_VERTICAL);
+        for (int i = 0; i < magicNumber; i++)
+            atb(new AttackDamageRandomEnemyAction(this, AbstractGameAction.AttackEffect.SLASH_VERTICAL));
+    }
+
+    @Override
+    public void upp() {
+        if (isBranchUpgrade())
+            branchUpgrade();
+        else
+            baseUpgrade();
+        baseSecondDamage = baseDamage;
+    }
+
+    public void baseUpgrade() {
+        upgradeDamage(2);
+    }
+
+    public void branchUpgrade() {
+        upgradeDamage(-2);
+        upgradeMagicNumber(1);
+        rawDescription = CardCrawlGame.languagePack.getCardStrings(ReboundVolley.ID).EXTENDED_DESCRIPTION[0];
+    }
+
+    @Override
+    public void calculateCardDamage(AbstractMonster mo) {
+        super.applyPowers();
```

Comment:
Usually you call the method you're overwriting at the start, then change variables then call the method again.
I'm not sure if applyPowers is the correct call here

```suggestion
        super.calculateCardDamage(mo);
```

Reply:
Wait, what are you even doing here? secondDamage is a custom variable and already scales like damage, what do you even need to change calculateCardDamage for? Furthermore, only overriding this method and not applyPowers means this change is only applied when the player has dragged the card and is about to play it.

Please explain

Reply:
So this might be a bit janky, but the aim here was to make secondDamage only change with relation to the player's powers, relics, etc. and not with the targeted monster's. Since secondDamage is used in the cardtext to show the "random enemy" damage, I felt like it makes more sense for it to not change when a specific enemy is targeted, since the true damage value may end up being different from that, so it provides a more accurate picture of how much damage the card will deal.

Reply:
Further to my point, a card like Rip and Tear doesn't change based on any enemy's Vulnerable - this happens anyway since it's set to target ALL_ENEMY - so I thought it made sense to match that here, given that the effects are similar.

Reply:
Ah, I see where you're coming from. This is an interesting consideration to make.

I hope you've tested this thoroughly because this seems like the type of code that could easily cause some weirdness.

Reply:
I have indeed.
Also, looking at AbstractCard's applyPowers and calculateCardDamage methods, they appear to be functionally identical - aside from the latter method checking the target monster - so there shouldn't be any unexpected interactions.
(That isn't to say that I haven't made _sure_ that there aren't, of course. Always better safe than sorry.)